### PR TITLE
feature/ERC-07-home-team

### PR DESCRIPTION
**Pull request by sofiaperezm**
# FooCamp - Escuela de Robotica

## Trello ticket
[User Story 07: Home_El equipo](https://trello.com/c/LMsZBDbq/19-user-story-07-homeel-equipo)

## Description
This PR contains:

- The `templates/teamSection.hbs` file for the team component on the home page.
- The `src/scss/layout/__teamSection.scss` file for the styles in the team component.
- The  `src/scss/abstracts/_placeholders.scss` to add the %button placeholder.

Also, the following files were modified:

- The `src/scss/main.scss` file to import the mentioned file in the item above.
- The `src/scss/abstracts/_variables.scss` to add several colors.

## To reproduce

1. Run the project
2. Open the `http://localhost:8080/team-section.html` page 
3. Check the `team` component

## Screenshots

Default viewport based on Figma properties
Captura de Pantalla 2021-09-09 a la(s) 6 23 58 p  m

Team component in extra-small devices 
Captura de Pantalla 2021-09-10 a la(s) 6 54 10 p  m

Team component in small devices
Captura de Pantalla 2021-09-10 a la(s) 6 54 46 p  m

Team component in medium devices
Captura de Pantalla 2021-09-10 a la(s) 6 55 03 p  m

Team component in large devices
Captura de Pantalla 2021-09-10 a la(s) 6 55 23 p  m

### CODE REVIEW

File: src/scss/layout/_teamSection.scss [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+            display: flex;
+            flex-direction: column;
+            align-items: center;
+        }
+    }
+
+    &__img {
+        width: 100%;
+        height: 230px;
+        border-radius: 16px 16px 0px 0px;
```
Comment (diegoamt) [inline src/scss/layout/_teamSection.scss:49]: replace 0px by just 0.

File: src/scss/layout/_teamSection.scss [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    &__cta {
+        border-radius: 30px;
+        background-color: var(--casablanca);
+        border-style: none;
+        display: flex;
+        @include fontStyle(var(--dodger-blue), 16px);
+        line-height: 18px;
+        align-items: center;
+        justify-content: center;
+        padding: 0.7rem 2.2rem;
```
Comment (diegoamt) [inline src/scss/layout/_teamSection.scss:83]: .7rem

File: src/scss/main.scss [hunk 3]
```diff
@@ -10,6 +10,7 @@
 // Layout
 
 @import 'layout/header';
+@import 'layout/teamSection';
```
Comment (diegoamt) [inline src/scss/main.scss:13]: don't use camelCase notation on filenames. Please rename it to `team-section.scss`. Same for the template.

**Review by diegoamt — CHANGES_REQUESTED** [review]

**Review by camilomejiag — APPROVED** [review]

File: src/scss/main.scss (hunk 3 above)
Comment (diegoamt) [reply src/scss/main.scss:13]: Please update this line to @import 'layout/team-section';
and please, rename the template to teamSection.hbs. In this case as the content type ID in contentful is with camelCase, handlebars will show the same name for the tamplate. sorry about the confusion ! :(

**Review by diegoamt — CHANGES_REQUESTED** [review]

File: src/scss/main.scss (hunk 3 above)
Comment (sofiaperezm) [reply src/scss/main.scss:13]: Got it. Will do :)

**Comment by sofiaperezm** [comment]
hi @diegoamt, @camilomejiag I just requested a re-revied due to I modified the HTML code, the component `teamCards` now iterates through content from `teamCard`. Thanks in advance!

File: src/scss/layout/_team-section.scss [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+            display: flex;
+            flex-direction: column;
+            align-items: center;
+        }
+    }
+
+    &__image {
+        width: 100%;
+        height: 230px;
+        border-radius: 16px 16px 0 0;
```
Comment (diegoamt) [inline src/scss/layout/_team-section.scss:53]: you can remove this `border-radius` using `overflow: hidden` in the `.team__card container`.

File: src/scss/layout/_team-section.scss [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+        }
+
+        @include breakpoint('xl') {
+            grid-template-columns: repeat(4, 1fr);
+        }
+    }
+
+    &__card {
+        
+        background: var(--white);
```
Comment (diegoamt) [inline src/scss/layout/_team-section.scss:37]: use `background-color `instead of `background`.

File: src/scss/layout/_team-section.scss [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+    }
+
+    &__cta {
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        @include fontStyle(var(--dodger-blue), 16px);
+        @extend %button;
+        padding: 0.8rem 2.2rem;
+        //margin-top: 1.6rem;
```
Comment (diegoamt) [inline src/scss/layout/_team-section.scss:85]: remove this comment

**Review by diegoamt — CHANGES_REQUESTED** [review]

File: src/scss/layout/_team-section.scss [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+        }
+
+        @include breakpoint('lg') {
+            grid-template-columns: repeat(3, 1fr);
+        }
+
+        @include breakpoint('xl') {
+            grid-template-columns: repeat(4, 1fr);
+        }
+    }
```
Comment (diegoamt) [inline src/scss/layout/_team-section.scss:33]: Awesome job implementing responsive !!! 🥇

File: src/scss/main.scss [hunk 8]
```diff
@@ -10,6 +11,7 @@
 // Layout
 
 @import 'layout/header';
+@import 'layout/team-section';
```
Comment (diegoamt) [inline src/scss/main.scss:14]: the styles for this component should be in the components folder !
Please update this and import from components: @import 'components/team-section';

**Review by diegoamt — CHANGES_REQUESTED** [review]

File: src/scss/main.scss (hunk 8 above)
Comment (sofiaperezm) [reply src/scss/main.scss]: OMG! sorry! 
will do!

**Review by camilomejiag — APPROVED** [review]

**Review by diegoamt — APPROVED** [review]

**Pull request merged**